README.md

```diff
@@ -52,13 +52,36 @@ export default {
 
 **Note:** Nuxt >= 2.15.0 is required
 
+## 💡 Vite Config
+
+You can pass Vite configurations and plugins into the `vite` entry of `nuxt.config`
+
+```js
+import ViteWindiCSS from 'vite-plugin-windicss'
```

Comment:
Downside of this approach, is that we load vite plug-ins when they are not needed and note in vite mode (prod webpack build and nuxt start) but seems good to document and update later when we supported better way

Reply:
Besides plugins, do you have any idea of working Vite config (resolve, base, or something?) for use to demonstrate here?

Reply:
Maybe externals? If not we can also keep it empty just documenting how to pass if needed

Reply:
I'll leave it empty for now. Thanks!